settings.gradle

```diff
@@ -28,8 +28,8 @@ if (!System.getenv().containsKey("SUB_BUILD")) {
 } else {
     def subBuild = System.getenv().get("SUB_BUILD")
```

Comment:
Now that this is getting used in more places, it might be nice to move it up a level in visibility, so the below doesn't have to repeat itself much.

Reply:
Could you please clarify? This sub-build is only defined in this settings file and is already at top level of the file.

Reply:
I meant you could have move the variable definition  `def subBuild = System.getenv().get("SUB_BUILD")` to higher up in the file and then used it below instead of `System.getenv().containsKey("SUB_BUILD")` around line 79 below. Not a huge deal.